gpkitmodels/GP/aircraft/wing/sparloading.py

```diff
@@ -16,24 +16,27 @@ class SparLoading(Model):
     Nsafety         1.0            [-]     safety load factor
     kappa           0.2            [-]     max tip deflection ratio
     W                              [lbf]   loading weight
+    N                              [-]     loading factor
     twmax           30.*pi/180     [-]     max tip twist
+    Stip            1e-10          [N]     tip loading
+    Mtip            1e-10          [N*m]   tip moment
+    throot          1e-10          [-]     root deflection angle
+    wroot           1e-10          [m]     root deflection
+
+    Variables of length wing.N
+    --------------------------
+    q                       [N/m]   distributed wing loading
+    S                       [N]     shear along wing
+    M                       [N*m]   wing section root moment
+    th                      [-]     deflection angle
+    w                       [m]     wing deflection
 
     Variables of length wing.N-1
     ----------------------------
-    Mr                      [N*m]   wing section root moment
-    M                       [N*m]   local moment
+    Mtw                     [N*m]   local moment due to twisting
     theta                   [-]     twist deflection
-
-    Upper Unbounded
```

Comment:
why'd you remove all the bounds? :p

Reply:
I removed the bounds because it got really annoying as I was debugging

Reply:
Do you think they're a useful feature overall? (during dev you can add SKIP VERIFICATION to disable checking)